polls/tool/src/bundle/org/sakaiproject/poll/bundle/Messages.properties

```diff
@@ -111,6 +114,7 @@ error_tofew_votes.voteCollection=You have selected too few options. (Minimum: {0
 error_exact_required.voteCollection=You must select the required number of options. (Required: {0})
 error_novote.voteCollection=You have selected no options. You must select at least {0}
 error_no_text=Your poll must have a question
+error_batch_options=You have to provide a valid file to create options massively. Ensure the file size is accepted and is a valid Excel or CSV file.
```

Comment:
Depending on the wording that is chosen above, update this "create options massively" wording too.

Reply:
good